OpenRA.Mods.Common/Activities/Hunt.cs

```diff
@@ -23,10 +23,11 @@ public class Hunt : Activity
 
 		public Hunt(Actor self)
 		{
-			var attack = self.Trait<AttackBase>();
+			var attack = self.TraitsImplementing<AttackBase>();
 			targets = self.World.ActorsHavingTrait<Huntable>().Where(
 				a => self != a && !a.IsDead && a.IsInWorld && a.AppearsHostileTo(self)
```

Comment:
No I mean why do we need to check self != a if we later check if self is hostile to a?
Can an actor ever be hostile to himself?